Clean up rpmlint warnings, fix automatic dependencies

flowerysong opened the pull request.

flowerysong committed:
Clean up rpmlint warnings, fix automatic dependencies

kyrias commented:
Hey, sorry for breaking this now, heh!  Any chance you could update the spec for meson?

flowerysong committed:
Update spec for meson

Untested, since I don't have an RPM-based system with meson available.
It's more likely to work than calling no-longer-existent files, though.

flowerysong commented:
I updated the spec per https://fedoraproject.org/wiki/Packaging:Meson, but I can't easily test it. We don't actually build yasl packages; we were thinking about it when I did the initial RPM packaging, and I had these fixes sitting around so I opened a pull request, but in production we just embed yasl in our projects.

kyrias commented:
Ah I see, thanks for the changes anyway.  I think I'll merge it and then spin up a machine in the next few days to test if it works.  Even if it doesn't work, it would have been broken without the change as well.
